mlflow/langchain/databricks_dependencies.py

```diff
@@ -16,18 +16,22 @@
 def _extract_databricks_dependencies_from_retriever(
     retriever, dependency_dict: DefaultDict[str, List[Any]]
 ):
-    from langchain.embeddings import DatabricksEmbeddings
-    from langchain.vectorstores import DatabricksVectorSearch
+    from langchain.embeddings import DatabricksEmbeddings as OldDatabricksEmbeddings
```

Comment:
I suspect that LangChain will remove `DatabricksEmbeddings` in a future release. Can we try/catch the `OldDatabricksEmbeddings` import & checks?